upi/openstack/04_control-plane.yaml

```diff
@@ -29,18 +29,18 @@
       cmd: "openstack port set --tag {{ cluster_id_tag }} {{ item.1 }}-{{ item.0 }}"
     with_indexed_items: "{{ [os_port_master] * os_cp_nodes_number }}"
 
-  - name: 'Create the second Control Plane ports'
+  - name: 'Create the ACI containers Control Plane ports'
```

Comment:
Similarly here based on previous comments and for the following changes

Reply:
this one is fine (i was using opflex terminology before but i changed it)